nextflow_schema.json

```diff
@@ -508,6 +508,52 @@
             },
             "fa_icon": "fas fa-file-signature"
         },
+        "function_interproscan": {
+            "title": "Function: INTERPROSCAN",
+            "type": "object",
+            "description": "Functionally annotates all annotated coding regions.",
+            "default": "",
+            "properties": {
+                "run_function_interproscan": {
+                    "type": "boolean",
+                    "description": "Activates the functional annotation of annotated coding regions.",
+                    "help_text": "Activates the annotation of annotated coding regions using [InterProScan](https://github.com/ebi-pf-team/interproscan). These annotate the locus tags to protein and domain levels according to the InterPro database. \n\nMore details can be found in the tool [documentation](https://interproscan-docs.readthedocs.io/en/latest/index.html)."
+                },
+                "function_interproscan_db_url": {
+                    "type": "string",
+                    "default": "http://ftp.ebi.ac.uk/pub/software/unix/iprscan/5/5.67-99.0/interproscan-5.67-99.0-64-bit.tar.gz",
+                    "help_text": "This allows the user to change the default InterProScan version for coding region functional annotation, which allows for the latest release usage. By default this is set to `http://ftp.ebi.ac.uk/pub/software/unix/iprscan/5/5.67-99.0/interproscan-5.67-99.0-64-bit.tar.gz`. ",
+                    "description": "Change the database version used for annotation."
+                },
+                "function_interproscan_db": {
+                    "type": "string",
+                    "help_text": "Use this to supply the path to a pre- downloaded InterProScan database. This can be any unziped InterProScan version. By default the database version '5.67-99.0' is downloaded. \n\nFor more details on where to find different InterProScan databases see tool [documentation](https://interproscan-docs.readthedocs.io/en/latest/UserDocs.html#obtaining-a-copy-of-interproscan).\n",
```

Comment:
```suggestion
                    "help_text": "Use this to supply the path to a pre-downloaded InterProScan database. This can be any unzipped InterProScan version. By default the database version '5.67-99.0' is downloaded. \n\nFor more details on where to find different InterProScan databases see tool [documentation](https://interproscan-docs.readthedocs.io/en/latest/UserDocs.html#obtaining-a-copy-of-interproscan).\n",
```